tests/common/devices/multi_asic.py

```diff
@@ -567,3 +567,30 @@ def get_internal_bgp_peers(self):
                 config_facts.get("BGP_INTERNAL_NEIGHBOR", {})
             )
         return bgp_internal_neighbors
+
+    def docker_cmds_on_all_asics(self, cmd, container_name):
+        """This function iterate for ALL asics and execute cmds"""
+        duthost = self.sonichost
+        if duthost.is_multi_asic:
+            for n in range(duthost.facts['num_asic']):
+                container = container_name + str(n)
+                self.shell(argv=["docker", "exec", container, "bash", "-c", cmd])
+        else:
+            self.shell(argv=["docker", "exec", container_name, "bash", "-c", cmd])
+
+    def docker_copy_to_all_asics(self, container_name, src, dst):
+        """This function copy from host to ALL asics"""
+        duthost = self.sonichost
+        if duthost.is_multi_asic:
+            for n in range(duthost.facts['num_asic']):
+                container = container_name + str(n)
+                self.shell("sudo docker cp {} {}:{}".format(src, container, dst))
+        else:
+            self.shell("sudo docker cp {} {}:{}".format(src, container_name, dst))
+
+    def docker_copy_from_asic0(self, container_name, src, dst):
```

Comment:
suggest to rename this to "docker_copy_from_asic" and change the func signature to have an asic parameter and have 0 as the default.  In that way, in future if there is a need to copy from other asics, this function can be used
docker_copy_from_asic(self, container_name, src, dest, asic=0):